app/util/PertaxValidators.scala

```diff
@@ -63,7 +63,7 @@ object PertaxValidators {
         value.fold[Map[String, String]](Map.empty)(v => Map(key -> v))
     })
 
-  private val AddressLineRegex = """^[ A-Za-z0-9&’'(),-./sôéàëŵŷáîïâêûü]+""".r
+  private val AddressLineRegex = """^[A-Za-z0-9 \-,.&'\/]+""".r
```

Comment:
Can we replace regex to be  **^[A-Za-z0-9 \\-,.&'\\/]{1,35}$**?

Reply:
35 character length validation is already there in AddressDto.scala. The reason behind having separate validations for allowed character set and 35 character length is that we need to show two separate error message to the user for respective scenarios.